architecture/smg/02-smg-design.md

```diff
@@ -83,12 +90,27 @@ SSH tunnel from a port on the local machine. There is an excellent paid tool for
 MacOS called [Core Tunnel](https://coressh.io/).
 This allows for easy configuration of tunnels and ensures that they stay open.
 
+Core tunnel settings are default, apart from
+
+General:
+
+* Host: bastion-datalabs.nerc.ac.uk, port 22
+* User: ubuntu
+* Forward: localhost:6443 to datalabs-k8s-master:6443
+
+Connection:
+
+* Private Key: ~/.ssh/<ssh_private_key>
+* Forward Agent: yes
+
+Once connected, you can tick 'Automatically connect on startup'.
+
 On Linux these are configured using the SSH config in the same way as the SSH connections
 but to ensure that they stay open a utility called AutoSSH can be used. To configure the
 tunnel add another block to the SSH config. An example for the Kubernetes API:
 
 ```bash
-Host k8s-tunnel
+Host nerc-k8s-tunnel
```

Comment:
Is it worth this being `datalabs-k8s-tunnel` to be consistent with the other naming?

Reply:
I've followed Ola's naming convention to keep things consistent.